lte/protos/apn.proto

```diff
@@ -80,17 +82,15 @@ message APNConfiguration {
 // Network wide APN configuration
 message APNConfig {
   uint32 id = 1;
-  // Contains either the APN Name or wildcard "*"
-  string service_selection = 2;
```

Comment:
why did you remove the service_selection? This is used to name the APN (e.g., "magma.ipv4", "ims", "internet", etc.)

Reply:
Earlier APNConfig was housed within a subscriber and it made sense to name the APN. This is the new config model for APN which is independently streamed down, the service_selection attribute needn't be a part of the APN config. In subscriber config, I added a new attribute
```
message SubscriberAPNConfig {
  uint32 apn_id = 1;
  string assigned_static_ip = 2;
}
```
subscriber_apn_config . This does the apn selection and also provides any additional APN specific subscriber configuration.

Reply:
let's not change the field names such as  service_selection or context_id otherwise we will lose parity on s6a interface in federated vs. non-federated cases.

Reply:
currently subscriberdb (in non-federated case) generates the context_id (uint32), but actually it makes more sense for the orc8r to assign it as well.